content/en/releases/patch-releases.md

```diff
@@ -85,6 +85,14 @@ releases may also occur in between these.
 
 ## Detailed Release History for Active Branches
 
+### 1.23
+
+**1.23** enters maintenance mode on **2022-12-28**.
+
+End of Life for **1.23** is **2023-02-28**.
+
+No patch releases.
```

Comment:
```suggestion
Patch release schedule to be determined.
```
Is it better to state that the 1.23 patch schedule is to be determined to deter an impression of no patch releases for 1.23